docs/source/02-Analytics-Query/Funnel-View/Sign-Up-Funnel.sql

```diff
@@ -0,0 +1,334 @@
+/*
+**Question**:
+
+There are 8 primary phases in login.gov sign up flow when user comes from a service provider.
+
+1. User entered a valid email
+2. User entered a confirmation email
+3. User created a valid password
+4. User visit the MFA setup page
+5. User finished MFA setup
+6. User viewed personal key page
+7. visit Handoff page with service provider
+8. Sign up workflow finished, and land to service provider
+
+We want to know how many unique sessions are there in each phase.
+
+**Definition**:
+
+1. User entered a valid email::
+
+    if have 'User Registration: Email Submitted' and success = True
+        return True
+
+2. User entered a confirmation email::
```

Comment:
I asked this question months ago to end user once. They confirmed that a user id will be created after email entered, even though it could be a dummy email.

We used to clean up those user_id associated with not-confirmed email regularly. But now we don't do that.

So dummy email could be one of the reason, I will pull those not-confirmed email from redshift to validate my assumption. I will let you know when I finish.